### ISSUE TITLE
🐞 "Getting Started" fails at Step 2

### ISSUE THREAD

**Issue by blaisep** [dagger/hello-dagger#3]
### What is the issue?

The python track fails in the second lesson (Test the Application)
Traceback at: https://paste.ofcode.org/pKgDXLDxK8u7V64xgWk8eS

### Log output

https://paste.ofcode.org/pKgDXLDxK8u7V64xgWk8eS

### Steps to reproduce

`dagger run python ci/main.py`

using dagger v0.8.5 darwin/amd64

### SDK version

dagger-io          0.8.5

### OS version

darwin/amd64

**Comment by helderco** [comment]
You mean the [Quickstart: Test the Application](https://docs.dagger.io/quickstart/947391/test) (steps not numbered). There's a different Python [Get Started: Step 2](https://docs.dagger.io/sdk/python/628797/get-started#step-2-create-a-dagger-client-in-python), so I was confused for a bit until I saw the traceback.

From the traceback I see your dir is `dagger-chatbot`. Are you using a different codebase from the one in the quickstart? If not, please check if you're running from the directory that has `package.json`. The `ci` dir should be next to `package.json`.

Also make sure you're running SDK version that is compatible with your CLI version.

**Comment by vikram-dagger** [comment]
I checked it as well just now, following the steps for Python. It seems to work fine. Output below.

```
$ dagger run python ci/main.py 
╮ 
▽ init
█ [4.50s] connect
┣ [3.40s] starting engine
┣ [0.31s] starting session
┻ 
█ [374.1s] python ci/main.py
┃ PASS src/App.test.tsx (7.259 s)                                                                                               
┃   ✓ renders learn dagger link (76 ms)                                                                                         
┃                                                                                                                               
┃ Test Suites: 1 passed, 1 total                                                                                                
┃ Tests:       1 passed, 1 total                                                                                                
┃ Snapshots:   0 total                                                                                                          
┃ Time:        11.586 s                                                                                                         
┃ Ran all test suites.                                                                                                          
┃ One of your dependencies, babel-preset-react-app, is importing the                                                            
┃ "@babel/plugin-proposal-private-property-in-object" package without                                                           
┃ declaring it in its dependencies. This is currently working because                                                           
┃ "@babel/plugin-proposal-private-property-in-object" is already in your                                                        
┃ node_modules folder for unrelated reasons, but it may break at any time.                                                      
┃                                                                                                                               
┃ babel-preset-react-app is part of the create-react-app project, which                                                         
┃ is not maintianed anymore. It is thus unlikely that this bug will                                                             
┃ ever be fixed. Add "@babel/plugin-proposal-private-property-in-object" to                                                     
┃ your devDependencies to work around this error. This will make this message                                                   
┃ go away.                                                                                                                      
┃                                                                                                                               
┣─╮ 
│ ▽ host.directory .
│ █ [1.42s] upload .
│ ┣ [0.85s] transferring eyJvd25lcl9jbGllbnRfaWQiOiIzamZ6aDRlYWQ4ZnpvZG9udDN1bzZnOHU0IiwicGF0aCI6Ii4iLCJpbmNsdWRlX3BhdHRlcm5zIjpudWxsLCJleGNsdWRlX3BhdHRlcm5zIjpudWxsLCJmb2xsb3dfcGF0aHMiOm51bGwsInJlYWRfc2luZ2xlX2ZpbGVfb25seSI6ZmFsc2UsIm1heF9maWxlX3NpemUiOjB9:
│ █ [0.50s] copy .
│ ┣─╮ copy .
│ ┻ │ 
┣─╮ │ 
│ ▽ │ from node:16-slim
│ █ │ [2.77s] resolve image config for docker.io/library/node:16-slim
│ █ │ [0.35s] pull docker.io/library/node:16-slim
│ ┣ │ [0.28s] resolve docker.io/library/node:16-slim@sha256:3ebf2875c188d22939c6ab080cfb1a4a6248cc86bae600ea8e2326aa03acdb8f
│ ┻ │ 
█◀──╯ [0.36s] copy / /src
█ [327.4s] exec docker-entrypoint.sh npm install
┃ npm WARN deprecated w3c-hr-time@1.0.2: Use your platform's native performance.now() and performance.timeOrigin.               
┃ npm WARN deprecated stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. S
┃ ee the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort
┃ #browser_compatibility                                                                                                        
┃ npm WARN deprecated sourcemap-codec@1.4.8: Please use @jridgewell/sourcemap-codec instead                                     
┃ npm WARN deprecated rollup-plugin-terser@7.0.2: This package has been deprecated and is no longer maintained. Please use @roll
┃ up/plugin-terser                                                                                                              
┃ npm WARN deprecated workbox-cacheable-response@6.6.0: workbox-background-sync@6.6.0                                           
┃ npm WARN deprecated @babel/plugin-proposal-private-methods@7.18.6: This proposal has been merged to the ECMAScript standard an
┃ d thus this plugin is no longer maintained. Please use @babel/plugin-transform-private-methods instead.                       
┃ npm WARN deprecated @babel/plugin-proposal-class-properties@7.18.6: This proposal has been merged to the ECMAScript standard a
┃ nd thus this plugin is no longer maintained. Please use @babel/plugin-transform-class-properties instead.                     
┃ npm WARN deprecated @babel/plugin-proposal-nullish-coalescing-operator@7.18.6: This proposal has been merged to the ECMAScript
┃  standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-nullish-coalescing-operator instead
┃ .                                                                                                                             
┃ npm WARN deprecated @babel/plugin-proposal-optional-chaining@7.21.0: This proposal has been merged to the ECMAScript standard 
┃ and thus this plugin is no longer maintained. Please use @babel/plugin-transform-optional-chaining instead.                   
┃ npm WARN deprecated @babel/plugin-proposal-numeric-separator@7.18.6: This proposal has been merged to the ECMAScript standard 
┃ and thus this plugin is no longer maintained. Please use @babel/plugin-transform-numeric-separator instead.                   
┃ npm WARN deprecated svgo@1.3.2: This SVGO version is no longer supported. Upgrade to v2.x.x.                                  
┃                                                                                                                               
┃ added 1394 packages, and audited 1395 packages in 5m                                                                          
┃                                                                                                                               
┃ 258 packages are looking for funding                                                                                          
┃   run `npm fund` for details                                                                                                  
┃ npm                                                                                                                           
┃ 6 high severity vulnerabilities                                                                                               
┃                                                                                                                               
┃ To address all issues (including breaking changes), run:                                                                      
┃   npm audit fix --force                                                                                                       
┃                                                                                                                               
┃ Run `npm audit` for details.                                                                                                  
┃  notice                                                                                                                       
┃ npm notice New major version of npm available! 8.19.4 -> 10.1.0                                                               
┃ npm notice Changelog: <https://github.com/npm/cli/releases/tag/v10.1.0>                                                       
┃ npm notice Run `npm install -g npm@10.1.0` to update!                                                                         
┃ npm notice                                                                                                                    
█ [37.2s] exec docker-entrypoint.sh npm test -- --watchAll=false
┃                                                                                                                               
┃ > hello-dagger@0.1.0 test                                                                                                     
┃ > react-scripts test --watchAll=false                                                                                         
┃                                                                                                                               
┃ PASS src/App.test.tsx (7.259 s)                                                                                               
┃   ✓ renders learn dagger link (76 ms)                                                                                         
┃                                                                                                                               
┃ Test Suites: 1 passed, 1 total                                                                                                
┃ Tests:       1 passed, 1 total                                                                                                
┃ Snapshots:   0 total                                                                                                          
┃ Time:        11.586 s                                                                                                         
┃ Ran all test suites.                                                                                                          
┃ One of your dependencies, babel-preset-react-app, is importing the                                                            
┃ "@babel/plugin-proposal-private-property-in-object" package without                                                           
┃ declaring it in its dependencies. This is currently working because                                                           
┃ "@babel/plugin-proposal-private-property-in-object" is already in your                                                        
┃ node_modules folder for unrelated reasons, but it may break at any time.                                                      
┃                                                                                                                               
┃ babel-preset-react-app is part of the create-react-app project, which                                                         
┃ is not maintianed anymore. It is thus unlikely that this bug will                                                             
┃ ever be fixed. Add "@babel/plugin-proposal-private-property-in-object" to                                                     
┃ your devDependencies to work around this error. This will make this message                                                   
┃ go away.                                                                                                                      
┃                                                                                                                               
┻ 
• Engine: 1afc8823a5e0 (version v0.8.4)
• Duration: 6m14.057s
⧗ 378.7s ✔ 23
```

**Comment by gerhard** [comment]
Closing this since there isn't anything new that we can follow-up on. Please re-open if this is still an issue @blaisep 👍
